config/short-url.php

```diff
@@ -148,4 +148,20 @@
     |
     */
     'validate_config' => false,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Database connection and tables
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the database connection and tables that get used
+    | while using this short url library. This connection is used when another
+    | is not explicitly specified when executing a given shortening function.
+    |
+    */
+    'database' => [
+        'urls_table' => 'short_urls',
+        'visits_table' => 'short_url_visits',
+        'connection' => 'default',
```

Comment:
I might be wrong, but I think this might need to be `null` rather than `'default'`? Then in the code, we'll let Laravel fallback and handle the default connection name?